packages/shared/src/domain/patient/demographics.ts

```diff
@@ -0,0 +1,17 @@
+import { z } from "zod";
+import { createNonEmptryStringSchema } from "../../common/string";
+import { dobSchema } from "../dob";
+import { genderAtBirthSchema } from "../gender";
+import { personalIdentifierSchema } from "../identifier";
+import { addressSchema } from "../address";
+import { contactSchema } from "../contact";
+
+export const demographicsSchema = z.object({
+  firstName: createNonEmptryStringSchema("firstName"),
+  lastName: createNonEmptryStringSchema("lastName"),
+  dob: dobSchema,
+  genderAtBirth: genderAtBirthSchema,
+  personalIdentifiers: z.array(personalIdentifierSchema).nullish(),
```

Comment:
in theory I thought this was going to be an empty array from the server, but was getting zod schemas while testing, so `nullish()` is the most lenient option here.

Generally I think returning `null` is confusing, and would advocate for either always a list (that can be empty) or a non-empty list of undefined.